### PR TITLE
scr-1050

### PR DESCRIPTION
**Pull request by thet**
This PR is broken down in individual commits. For the new upgrade steps in recensio.policy we can just use the commits related to that.

Related PRs:
- https://github.com/syslabcom/recensio.buildout/pull/1
- https://github.com/syslabcom/recensio.policy/pull/1
- https://github.com/syslabcom/recensio.theme/pull/3
- https://github.com/syslabcom/recensio.translations/pull/1

### CODE REVIEW

**Review by reinhardt — CHANGES_REQUESTED** [review]
I'm not quite comfortable with upgrading to a new Plone version, even a minor one, without knowing exactly what will change, because we have customizations that may rely on internals. I'd suggest splitting off the Plone upgrade to a separate PR and tackle that later.

File: profiles/base.cfg [hunk 1]
```diff
@@ -208,31 +212,14 @@ recipe = collective.recipe.omelette
 eggs = ${instance:eggs}
 ignores = rdflib
 
-[ipzope]
```
Comment (reinhardt) [inline profiles/base.cfg:211]: Any particular reason why you removed ipzope?

Comment (thet) [reply profiles/base.cfg:211]: actually, no. It was at a point where I wanted to create a personal buildout.
should I re-add it?

**Comment by thet** [comment]
I removed the Plone 4.3.20 upgrade and added in a seperate branch: https://github.com/syslabcom/recensio.buildout/pull/2

File: profiles/base.cfg (hunk 1 above)
Comment (reinhardt) [reply profiles/base.cfg:211]: I've done so locally, I'll commit that. Cheers!

**Review by reinhardt — APPROVED** [review]

**Pull request merged**